### PR TITLE
EDSC-2834: Adds harmony download links

### PR DESCRIPTION
**Pull request by trevorlang**

### CODE REVIEW

File: static/src/js/components/CollectionDetails/CollectionDetailsBody.js [hunk 1]
```diff
@@ -267,14 +267,15 @@ export const CollectionDetailsBody = ({
                       </dt>
                       <dd>
                         {
-                          Object.keys(reformattings).map((supportedInputFormat) => {
+                          Object.keys(reformattings).map((supportedInputFormat, i) => {
```
Comment (trevorlang) [inline static/src/js/components/CollectionDetails/CollectionDetailsBody.js:270]: Add comment

File: static/src/js/components/EDSCTabs/__tests__/EDSCTabs.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from 'react'
```
Comment (trevorlang) [inline static/src/js/components/EDSCTabs/__tests__/EDSCTabs.js:1]: Change filename

File: static/src/js/components/ProgressRing/ProgressRing.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+export const ProgressRing = ({
+  className,
+  progress,
+  strokeWidth,
+  width
+}) => {
+  // Calculate the radius for the current width and stroke width
+  const radius = (width / 2) - strokeWidth
+
+  const circumference = radius * 2 * Math.PI
```
Comment (trevorlang) [inline static/src/js/components/ProgressRing/ProgressRing.js:22]: Parens

File: static/src/js/components/ProgressRing/ProgressRing.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+  // Calculate the offset to be used on the stroke
+  const offset = circumference - progress / 100 * circumference
```
Comment (trevorlang) [inline static/src/js/components/ProgressRing/ProgressRing.js:25]: Parens

File: static/src/js/components/ProjectPanels/ProjectPanelSection.js [hunk 5]
```diff
@@ -8,26 +9,77 @@ import './ProjectPanelSection.scss'
  * @param {object} props - The props passed into the component.
  * @param {node} props.children - The section children.
  * @param {object} props.heading - The text to be used for the section heading.
+ * @param {object} props.headingLevel - A custom heading level. Defaults to 'h3'
```
Comment (trevorlang) [inline static/src/js/components/ProjectPanels/ProjectPanelSection.js:12]: Update type

File: static/src/js/components/ProjectPanels/VariableTreePanel.scss [hunk 6]
```diff
@@ -5,58 +5,50 @@
...
-      color: $color__black--500;
-
-      &:disabled {
-        background-color: $color__black--100;
-        color: $color__black--700;
-        opacity: 1;
-        cursor: pointer;
-      }
+  }
+  &__tree-switcher-buttons {
```
Comment (trevorlang) [inline static/src/js/components/ProjectPanels/VariableTreePanel.scss:9]: Add empty lines

File: static/src/js/components/TextWindowActions/TextWindowActions.js [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+  }
+
+  const handleClose = () => {
+    setShowModal(false)
+  }
+
+  const copyToClipboard = () => {
+    if (textareaElRef.current) {
+      textareaElRef.current.select()
+      textareaElRef.current.select()
```
Comment (trevorlang) [inline static/src/js/components/TextWindowActions/TextWindowActions.js:46]: Remove

File: test-env.js [hunk 8]
```diff
@@ -1,3 +1,5 @@
+// import { JSDOM } from 'jsdom'
```
Comment (trevorlang) [inline test-env.js:1]: Remove

**Review by abbottry — APPROVED** [review]

**Review by macrouch — APPROVED** [review]

**Pull request merged**